wp-content/themes/pub/wporg-learn-2020/css/components/_workshop-page.scss

```
@@ -28,25 +30,24 @@
 
 		@include breakpoint( $breakpoint-tablet ) {
 			padding-top: 0;
-			padding-left: 16px;
+			padding-left: $gutter-default;
 		}
 	}
 
 	&_list {
 		margin-top: $gutter-default * 3;
-		color: $color__text;
+	}
 
-		h2 {
-			font-size: 24px;
-		}
+	h2 {
+		font-size: 24px;
+	}
 
-		ol {
-			margin: 0 0 0 16px;
-			padding: 0;
-		}
+	ol, ul {
+		margin: 0 0 0 18px;
+		padding: 0;
+	}
 
-		li {
-			margin-top: $gutter-default / 2;
-		}
+	li {
+		margin-top: $gutter-default / 2;
```

Look like this is adding an extra space above the list too, what about

```suggestion
	li + li {
		margin-top: $gutter-default / 2;
```

---

Oh neat. 💯